charts/posthog/tests/redis.yaml

```diff
@@ -0,0 +1,63 @@
+suite: Redis
```

Comment:
If we call the suite `Redis` I think we should include here all the possible rendered values used by the chart. 
If we call this `Plugins deployment definition` I think it could work (despite we don't test a lot of other stuff related to it)

Reply:
Iterating on this comment: why do we have `REDIS_URL` only for the plugins deployment and not for events/migrate/beat/web and workers?

Reply:
I'm not sure how we use Redis exactly, but I suspect we should have it set for web & events & beat too based on https://posthog.com/docs/self-host/architecture.